mobile-center/package.json

```diff
@@ -16,7 +16,7 @@
   "bugs": {
     "url": "https://github.com/microsoft/mobile-center-sdk-react-native/issues"
   },
-  "homepage": "https://github.com/microsoft/mobile-center-sdk-react-native#readme",
+  "homepage": "https://github.com/microsoft/mobile-center-sdk-react-native/mobile-center#readme",
```

Comment:
This link won't work, it does not have a a branch or a tag.

For example to link develop readme: https://github.com/Microsoft/mobile-center-sdk-react-native/blob/develop/README.md